apps/ledger-live-desktop/tests/utils/fileUtils.ts

```diff
@@ -0,0 +1,18 @@
+import crypto from "crypto";
+import { appendFile } from "fs/promises";
+
+export function generateUUID(): string {
+  return crypto.randomBytes(16).toString("hex");
+}
+
+export function appendFileErrorHandler(e: Error | null) {
```

Comment:
It is not used elsewhere than below if I'm not mistaken. Do we need to keep as a function ?

Reply:
indeed, moved it inside safeAppendFile